docs/pages/docs/production/railway.mdx

```diff
@@ -35,6 +35,10 @@ yarn start --schema $RAILWAY_DEPLOYMENT_ID
 npm run start -- --schema $RAILWAY_DEPLOYMENT_ID
 ```
 
+```bash [bun]
+bun start -- --schema $RAILWAY_DEPLOYMENT_ID
```

Comment:
Worth confirming this is correct, the extra `--` is not required for pnpm or yarn, only npm